graph/db/graph.go

```diff
@@ -471,28 +471,31 @@ func (c *ChannelGraph) ForEachChannel(cb func(*models.ChannelEdgeInfo,
 
 		// Load edge index, recombine each channel with the policies
 		// loaded above and invoke the callback.
-		return kvdb.ForAll(edgeIndex, func(k, edgeInfoBytes []byte) error {
-			var chanID [8]byte
-			copy(chanID[:], k)
-
-			edgeInfoReader := bytes.NewReader(edgeInfoBytes)
-			info, err := deserializeChanEdgeInfo(edgeInfoReader)
-			if err != nil {
-				return err
-			}
+		return kvdb.ForAll(edgeIndex,
+			func(k, edgeInfoBytes []byte) error {
```

Comment:
nit: `edgeIndex` should also be on this line. And closing parenthesis below aren't fully correct either.